<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>news_ope_teacher</class>
 <widget class="QWidget" name="news_ope_teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>562</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>562</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>公告详情</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#news_ope_teacher{
background-color: rgb(235, 244, 234);
}
.QPushButton:!hover
{
background-color: rgb(162, 210, 152);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);
border-radius:3px;
 padding: 3px;
}
.QPushButton:hover
{
background-color: rgb(85, 170, 127);
color: rgb(255, 255, 255);
font: 9pt &quot;微软雅黑&quot;;
border-color: rgb(255, 255, 255);
border-radius:3px
}
.QPushButton:pressed {
    padding-left: 5px;
    padding-top: 5px;
}</string>
  </property>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="titlelabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>9</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>公告标题</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="collectBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>收藏</string>
   </property>
  </widget>
  <widget class="QLabel" name="timelabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deplabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>部门</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>501</width>
     <height>341</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>textBrowser</zorder>
  <zorder>exitBtn</zorder>
  <zorder>titlelabel</zorder>
  <zorder>collectBtn</zorder>
  <zorder>timelabel</zorder>
  <zorder>deplabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
